packages/plugin-list/List.ts

```diff
@@ -108,60 +108,69 @@ export class List extends JWPlugin {
     renderToDom(currentContext: DomRenderingContext): [DomRenderingContext, DomRenderingMap] {
         let context = { ...currentContext };
         const node = context.currentNode;
-        const isUnrenderedListItem =
-            node.parent && node.parent.is(ListNode) && !VDocumentMap.toDom(node);
         let result;
-        if (isUnrenderedListItem) {
+        if (node.parent && node.parent.is(ListNode) && !this._isRenderingLi) {
+            this._isRenderingLi = true;
             result = this.renderListItemToDom({ ...context });
             context = result[0];
-        }
-        if (node.is(ListNode)) {
+            this._isRenderingLi = false;
```

Comment:
By "remove list rendering hack", you mean "replace it by another hack", right ? 🚎

Reply:
![image](https://user-images.githubusercontent.com/24205914/73541293-58609680-4432-11ea-81f9-b85afc8ccf15.png)